packages/react-provider/src/hooks/useProvider.ts

```diff
@@ -40,6 +40,24 @@ type ProviderEventPayload =
   | ProviderChainChangedEventPayload
   | ProviderInitiatedEventPayload
 
+type ProviderType = {
```

Comment:
why did u create extra interface if u got almost the same in the provider package?

Reply:
fixed